mode/python/python.js

```diff
@@ -249,15 +249,24 @@
     }
 
     function tokenLexer(stream, state) {
+      var sol = stream.sol();
+      if (sol){
+          state.whiteSpaceAtBeginningOfLine = true;
+      }
```

Comment:
The project uses two spaces for indentation. And you can directly put the call to `sol()` in the condition, no need for a variable.